src/layouts/components/Editor/components/MenuBar.tsx

```diff
@@ -233,22 +269,30 @@ export const MenuBar = ({ editor }: { editor: Editor }) => {
       type: "divider",
     },
     {
-      type: "item",
-      icon: BiTable,
-      title: "Add table",
-      action: () =>
-        editor
-          .chain()
-          .focus()
-          // NOTE: Default to smallest multi table
-          .insertTable({ rows: 3, cols: 3, withHeaderRow: true })
-          .run(),
-    },
-    {
-      type: "item",
-      icon: BiMinus,
-      title: "Divider",
-      action: () => editor.chain().focus().setHorizontalRule().run(),
+      type: "detailed-list",
+      label: "Add complex blocks",
+      icon: BiPlus,
+      items: [
+        {
+          name: "Accordion",
+          description: "Let users hide or show content.",
+          icon: EditorAccordionImage,
+          action: () => editor.chain().focus().setDetailsGroup().run(),
```

Comment:
i seem to have an error for `setDetailsGroup` - i think this might be due to a misplaced commit? (need tiptap pro integration i think)

Reply:
uhm need to run a 'npm i' after adding the env var, the deets are in 1pw!